source/modes/MenuModeConfigureSeats.cpp

```diff
@@ -160,12 +160,7 @@ void MenuModeConfigureSeats::activate()
                 CEGUI::ListboxTextItem* item = new CEGUI::ListboxTextItem(faction, cptFaction);
```

Comment:
Yep. We played 2 games (me and @akien-mga). No segfault. We played "The bridge" but we found that gold is almost only in a shared part of the map. That means that you end up fighting to death with your 5 first creatures ^^